sdk/src/test/java/me/uport/sdk/AccountStorageTest.kt

```diff
@@ -45,22 +47,21 @@ class AccountStorageTest {
                 ""
         )
 
-        storage.upsert(account)
+        storage.upsert(refAccount)
+        storage.upsert(refAccount)
```

Comment:
`storage.upsert(refAccount)` is called twice

Reply:
good catch, it used to be called twice before too and I missed it during my refactor